frontend/include/chpl/framework/error-classes-list.h

```diff
@@ -85,3 +85,5 @@ ERROR_CLASS(ProcTypeUnannotatedFormal,
 ERROR_CLASS(ProcDefExplicitAnonFormal,
     const uast::Function*,
     const uast::Formal*)
+ERROR_CLASS(SuperFromTopLevelModule,
+    const uast::AstNode*)
```

Comment:
Could you also accept a `Module*` here, and report its name in the error? i.e.,

```
Invalid use of 'super' in 'import' or 'use'; 'myModule' is a top-level module.
    |
 10 | import this.super.super;
    |
Module `myModule` was declared at the top level here:
    |
  1 | module MyModule {
    |
```
Furthermore, do you have information to report specifically if it's a `use` or `import`? We've been trying to do that. That second "Module myModule was declared"... could be a note.